src/EditorFeatures/Core/Classification/Semantic/AbstractSemanticOrEmbeddedClassificationViewTaggerProvider.cs

```diff
@@ -59,13 +59,7 @@ protected sealed override ITaggerEventSource CreateEventSource(ITextView textVie
 
         // Note: we don't listen for OnTextChanged.  They'll get reported by the ViewSpan changing and also the
         // SemanticChange notification. 
-        // 
-        // Note: because we use frozen-partial documents for semantic classification, we may end up with incomplete
```

Comment:
>frozen [](http://example.com/codeflow?start=32&length=6)

Don't see the GetDocumentWithFrozenPartialSemantics in this file either.

Reply:
it happens in the service we call into.